makefiles/e2e.mk

```diff
@@ -17,7 +17,20 @@ e2e-setup-core-wo-auth:
 
 .PHONY: e2e-setup-core-w-auth
 e2e-setup-core-w-auth:
-	helm upgrade --install --create-namespace --namespace vela-system --set image.pullPolicy=IfNotPresent --set image.repository=vela-core-test --set applicationRevisionLimit=5 --set dependCheckWait=10s --set image.tag=$(GIT_COMMIT) --wait kubevela ./charts/vela-core --set authentication.enabled=true --set authentication.withUser=true --set authentication.groupPattern=* --set featureGates.gzipResourceTracker=true
```

Comment:
We're missing the gzip test now

Reply:
Since we only have three possible compressions, I added the gzip test to another e2e target, for now. If we have other compression methods in the future, we may use other methods.

Reply:
I suggest not doing so. Leaving the e2e-setup-core-wo-auth without any compression method. It is okay to omit one of the compression tests in the e2e test.

Reply:
I will remove it. But I wonder is there any specific reason for having no compression in `e2e-setup-core-wo-auth`?

Reply:
By default, we do not switch on the ResourceTracker compression (neither zstd nor gzip). So in general, for most existing KubeVela users, they are using ResourceTracker without compression, and this might persist for some time (until we switch on the compression in the future, but there are downstream codes affected by this, like vela-prism). Although logically it is okay to set up the tests all with compression, I think it is also necessary to keep tests without compression to align with the real usage. The `e2e-setup-core-w-auth` (the name is actually confusing now), enforces some advanced features in multi-cluster e2e test. So it is okay to put new features and mechanism in this test.